File: censor/censor.py
fin = open('censor.in')
data = fin.readlines()
fin.close()
S = data[0][:-1]
T = data[1][:-1]

while T in S:
	S = S.replace(T, '')

fout = open('censor.out', 'w')
fout.write(S)
fout.close()

File: censor/censor.cpp
#include <fstream>
#include <string>
#include <regex>
using namespace std;

int main(void) {
	ifstream fin;
	fin.open("censor.in");
	string S, T;
	fin>>S>>T;
	fin.close();

	/*int counter = 0;
	for (int i = 0; i<S.length(); i++) {
		if (S[i] = T[counter]) {
			counter++;
		}
		else if (counter>0) {
			counter = 0;
		}
	}*/

	while (S.find(T)!=string::npos) {
		S = regex_replace(S, T, "");
	}

	ofstream fout;
	fout.open("censor.out");
	fout<<S;
	fout.close();

	return 0;
}
